app/javascript/components/modal/Modal.vue

```diff
@@ -15,18 +15,24 @@
             <p v-for="p in section.text" v-html="p"></p>
           </div>
 
-          <div v-if="index == 0" class="modal__bg-image item-margin--top item-margin--bottom relative" :style="{ 'background-image': `url(${data.modal.image})` }">
+          <div 
+            v-if="index == 0 && data.modal.image"
+            class="modal__bg-image item-margin--top item-margin--bottom relative"
+            :style="{
+              'background-image': `url(${data.modal.image})`,
+              'background-position': `${data.modal.image_position ? data.modal.image_position : '0% 0%'}`
```

Comment:
Yep, I've just put this into a computed property. Let me know if the syntax could be improved

<img width="600" alt="image" src="https://user-images.githubusercontent.com/56026099/120646547-8bc19400-c471-11eb-8420-a153b33a002d.png">

Reply:
Top - nice use of the es6 stuff :) I'm guessing there is no eslint set up on annual review. If it was, I think it would tell you:

- Let -> const
- Add a break line after the variable assignment (and before return, but here they're next to each other so only one needed)
- Remove unnecessary `;` after `this.data.modal`
- Lessen the indent on the closing bracket on the returned object so it aligns with the r of return

Reply:
Yep, that's all good. Made those changes you've suggested. Not sure if ESLint is set up on the project, but I would like to get that going ASAP as it's really good for catching some of these smaller issues.